### DISCUSSION TITLE
Δήλωση πρόθεσης προαιρετικής παρουσίασης

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#112] [Show and tell]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης προόδου της ομάδας σας στέλνοντας το τελικό αρχείο παρουσίασης

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ2 δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη ΟΣΣ

**Comment by iparthenios** [comment]
To team https://github.com/orgs/SDY60-2023/teams/hapodyni δηλώνει την πρόθεσή της για την προαιρετική παρουσίαση για σήμερα Κυριακή 3/12/2023 στις 18:00
Η παρουσίαση ετοιμάζεται και θ' αναρτηθεί σε λίγη ώρα...

**Comment by iparthenios** [comment]
[Παρουσίαση Ομάδας HApOdyNi.pdf](https://github.com/sdy60/syllabus/files/13539169/HApOdyNi.pdf)

**Comment by Maurphal** [comment]
Η ομάδα https://github.com/orgs/SDY60-2023/teams/islanders δηλώνει την πρόθεσή της για την προαιρετική παρουσίαση για σήμερα Κυριακή 3/12/2023 στις 18:00
[Παρουσίαση.pdf](https://github.com/sdy60/syllabus/files/13539527/default.pdf)

**Comment by IoannisTheodoropoulos** [comment]
Καλησπέρα σας, εκ μέρους της ομάδας https://github.com/orgs/SDY60-2023/teams/e-netteam δηλώνω την πρόθεσή μας να παρουσιάσουμε το μέρος Δ σήμερα 10/12/23

[eNetTeam.pdf](https://github.com/sdy60/syllabus/files/13627501/eNetTeam.pdf)


Ευχαριστούμε!
